crates/uv/src/commands/python/uninstall.rs

```diff
@@ -200,13 +200,13 @@ async fn do_uninstall(
         });
     }
 
-    let mut uninstalled = vec![];
+    let mut uninstalled: IndexSet<PythonInstallationKey> = IndexSet::default();
```

Comment:
Nit: I'd suggest `IndexSet::<PythonInstallationKey>::default()` to avoid duplicating the type.